### PR TITLE
Schema Foldering support

### LINKED ISSUE #111: Schema Foldering support
I'd like to be able to sort entity models into folders by their scheme name.

### PR DESCRIPTION
**Pull request by JohnGoldInc**
Enables Schema Foldering support #111

Closes #111

I had to change the  Constants.Connections.SqlServerConnection temporarily to run existing unit tests; Decided to check in my changes to fix the unit tests (but not the constant)

### CODE REVIEW

File: test/Scaffolding.Handlebars.Tests/ExpectedContexts.cs [hunk 1]
```diff
@@ -25,7 +28,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             if (!optionsBuilder.IsConfigured)
             {
 #warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
-                optionsBuilder.UseSqlServer(""Data Source=(localdb)\\MSSQLLocalDB; Initial Catalog=NorthwindTestDb; Integrated Security=True"");
+                optionsBuilder.UseSqlServer(""" + Constants.Connections.SqlServerConnection + @""");
```
Comment (tonysneed) [inline test/Scaffolding.Handlebars.Tests/ExpectedContexts.cs:31]: This breaks the unit tests, because the connection string in Constants needs two backslashes.

File: test/Scaffolding.Handlebars.Tests/ExpectedTypeScriptContexts.cs [hunk 2]
```diff
@@ -25,7 +27,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             if (!optionsBuilder.IsConfigured)
             {
 #warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
-                optionsBuilder.UseSqlServer(""Data Source=(localdb)\\MSSQLLocalDB; Initial Catalog=NorthwindTestDb; Integrated Security=True"");
```
Comment (tonysneed) [inline test/Scaffolding.Handlebars.Tests/ExpectedTypeScriptContexts.cs:28]: Please revert this change as well.

Comment (JohnGoldInc) [reply test/Scaffolding.Handlebars.Tests/ExpectedTypeScriptContexts.cs]: @tonysneed reverted

**Comment by tonysneed** [comment]
Thanks for the changes.  Would you please squash your commits? Here's how: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

After that run: `git push --force`
